Changes

```diff
@@ -84,6 +84,9 @@ Working version
 
 ### Compiler user-interface and warnings:
 
+- #11530: Include kinds in kind mismatch error message.
+  (Leonhard Markert, review by TODO)
```

Comment:
TODO is now Florian and me

Reply:
... déjà vu!

https://github.com/ocaml/ocaml/blob/ef8431146cb028c9d09db30bd9a327ee079d920b/Changes#L37-L38